views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+          not official languages of the country concerned, too).
+        </p>
+        <h3>
+          If you <em>only</em> want names
+        </h3>
+        <p>
+          For convenience, we isolate all the names of all the politicians
+          within each legislature and make them available in their own file.
+          (Like other files, look inside <code>countries.json</code> to find
+          its path and filename; in this case under <code>names</code>). This
+          file contains <code>name</code> and <code>id</code> fields. Note that
+          there may be more than one name for each politician (for example, we
+          might have the name of a Welsh politician rendered in English and
+          Chinese). If you need to consolidate them, use the <code>id</code> to
+          match them (or, as ever, use the JSON data instead).
+        </p>
+        <p>
+          In fact, we’ve even built an external service that consolidates all
+          the names into one CSV file (caution: that currently contains well
+          over 80,000 names and growing): see
+          <a href="https://github.com/everypolitician/everypolitician-names/">everypolitician-names</a>.
+        </p>
+        <h3>
+          IDs to other data sets, including WikiData
+        </h3>
+        <p>
+          We give each politician in EveryPolitician a unique ID (actually a
+          UUID). It’s keyed as <code>id</code> in the CSV. But often we’ll know
+          useful IDs for the same politician in other data sets too. You can
+          find these in the JSON data under <code>identifiers</code>.
+        </p>
+        <p>
+          Each identifier consists of an (<code>identifier</code>,
+          <code>scheme</code>) pair. For example, here’s an entry for the
+          identifier in WikiData (the database behind Wikipedia):
+        </p>
+        <pre>
+"identifiers": [
+  {
+    "identifier": "Q1091494",
+    "scheme": "wikidata"
+  }
+]
+</pre>
+        <p>
+          (Incidentally, you can view WikiData entries like this:
+          <a href="https://www.wikidata.org/wiki/Q1091494">wikidata.org/wiki/Q1091494</a>,
+          or using <a href="https://tools.wmflabs.org/reasonator/?q=Q1091494">the Reasonator</a>.
+          This example is the Parliament of Åland.)
+        <p>
+          We do this for other entities (such as legislatures themselves) as
+          well as politicians. Remember that external data sets might not be
+          complete, so it’s possible to have a useful identifier which isn’t
+          populated for <em>all</em> records. For example, WikiData only holds
+          data for entries that satisfy their “notability” criteria.
+        </p>
+        <p>
+          Some legislatures have adopted the good practice of assigning unique
+          IDs to their politicians, which can be useful when using their
+          official APIs. Whenever we can find IDs like this, we’ll add them as
+          identifiers in this way.
```

Comment:
or even something like "Whenever we're made aware of" that still pushes the assumption that we're expecting people to contribute.